glad/lang/c/loader/gl.py

```diff
@@ -171,6 +171,10 @@
 }
 #endif
 
+#if !defined(_WINDOWS_)
```

Comment:
Instead of check for `_WINDOWS_` we should set a flag when we define `APIENTRY` maybe something like `GLAD_DEFINED_APIENTRY` and when this is defined `undef` both symbols `APIENTRY` and `GLAD_DEFINED_APIENTRY`.

Reply:
Makes sense 👍